services-staticmap/src/main/java/com/mapbox/api/staticmap/v1/StaticMapCriteria.java

```diff
@@ -57,23 +57,23 @@
    *
    * @since 3.0.0
    */
-  public static final String LIGHT_STYLE = "light-v9";
+  public static final String LIGHT_STYLE = "light-v10";
 
   /**
    * Mapbox Dark style's a subtle, full-featured map designed to provide geographic context while
    * highlighting the data on your analytics dashboard, data visualization, or data overlay.
    *
    * @since 3.0.0
    */
-  public static final String DARK_STYLE = "dark-v9";
+  public static final String DARK_STYLE = "dark-v10";
 
   /**
    * Mapbox Satellite is our full global base map that is perfect as a blank canvas or an overlay
    * for your own data.
    *
    * @since 3.0.0
    */
-  public static final String SATELLITE_STYLE = "satellite-v9";
+  public static final String SATELLITE_STYLE = "satellite-v10";
```

Comment:
Whoops. The `mapbox/satellite-v*` "style" actually hasn't increased by one version (it's essentially a pass through to our `mapbox.satellite` raster tileset and as a result doesn't get updated often).

```suggestion
  public static final String SATELLITE_STYLE = "satellite-v9";
```

Reply:
Great, brought it back down to v9